### ISSUE TITLE
Can't delete save states

### ISSUE THREAD

**Issue by DrBoskonovitch** [geeksville/steamback#42]
I've been playing Baldur's Gate 3 and SteamBack has made about a dozen backups.  The problem is that BG3 saves can take upwards of 10 gigs.  SteamBack is now taking up more than 100 gigs.  I see no way to delete the saves in the plugin.

**Comment by geeksville** [comment]
yeah - good point.  I'll try to add such a button soon.  Though one thing helps for now is that it caps the snapshots at 10 (or 20 - I don't recall) backups (in total).  So only the most recent N backups are consuming space.

**Comment by geeksville** [comment]
alas, I really want to push out a build now to fix a more urgent bug (#38).  So I think I'll wait on adding this new feature, but I should add it in about a month.

**Comment by micah** [comment]
In the meantime, maybe let us know where these are saved on the filesystem, so we can go in and remove them manually.

**Comment by geeksville** [comment]
@micah I'm afk right now but (approxmately) they are in

/home/deck/homebrew/data/steamback/saves2

Alas I can't remember for sure if "data" is the name decky uses for that directory, but it is something approximately like that.  Then you can just rm -r saves2 to delete all saves.

**Comment by geeksville** [comment]
though to be clear: no need (typically) to remove them manually, because it
caps the number of saves at 10 and auto deletes any older than that.

**Comment by micah** [comment]
Thanks for that, i'll go poking around to see if I can find it there.

Do you know if I remove the steamback plugin, will it remove those? That
might be an easy quick solution.

**Comment by geeksville** [comment]
nope.  alas, removing plugins does not deinstall/delete any of their data
(which IMO is an incorrect choice but it is what the decky devs want to do).

**Comment by sucram65** [comment]
This was a good question. I installed Steam back but since I didn’t know where the files are going and there is still no option to delete the snaps I am probably going to uninstall it. It’s not good that it doesn’t clean itself up on the way out. Where exactly are the snaps stored? Please.

**Comment by sucram65** [comment]
I found it where you said it should be. Shame, because I like the idea. You’ve got to provide an option to delete the snaps in order to manage space.

**Comment by geeksville** [comment]
someday. ;-)

**Comment by geeksville** [comment]
or pull-requests are eagerly accepted ;-)
